modules/custom/az_google_tag/az_google_tag.install

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for az_google_tag module.
+ */
+
+use Drupal\Core\Url;
+
+/**
+ * Implements hook_requirements().
+ *
+ * @link https://github.com/az-digital/az_quickstart/issues/3248
+ */
+function az_google_tag_requirements($phase) {
+  $requirements = [];
+  if ($phase === 'runtime') {
+    $consent_mode_enabled = az_google_tag_consent_mode_enabled();
+    $tag_containers_url = Url::fromRoute('entity.google_tag_container.collection');
+    $requirements['az_google_tag'] = [
+      'title' => t('Google Tag consent mode setting(s)'),
+      'value' => t('Disabled'),
+      'description' => t(
+        'The "Enforce Privacy Consent Policy" setting is not currently enabled for any <a href="@url">Google Tag containers</a>. This is the recommended configuration for most UArizona websites at this time. Once a consent management platform is available for campus websites, the recommended setting(s) will likely change.',
+        ['@url' => $tag_containers_url->toString()]
+      ),
+      'severity' => REQUIREMENT_OK,
+    ];
+
```

Comment:
Question:  do we need to display anything if there aren't any containers with consent mode enabled?  Maybe it would be better to only display a warning if there are and display nothing if not?
```suggestion
```

Reply:
That seems reasonable to me.